app/code/Magento/Sales/Controller/Adminhtml/Order/ReviewPayment.php

```diff
@@ -50,21 +57,23 @@ public function execute()
                         }
                         break;
                     default:
-                        throw new \Exception(sprintf('Action "%s" is not supported.', $action));
+                        throw new \Magento\Framework\Exception\NotFoundException(
+                            __('Action "%1" is not supported.', $action)
+                        );
                 }
                 $this->orderRepository->save($order);
                 $this->messageManager->addSuccessMessage($message);
+                $resultRedirect->setPath('sales/order/view', ['order_id' => $order->getEntityId()]);
             } else {
                 $resultRedirect->setPath('sales/*/');
                 return $resultRedirect;
             }
+            // phpcs:ignore Magento2.Exceptions.ThrowCatch
```

Comment:
What is the reason for ignoring this rule?
I would recommend adding the `@throws` tags into DocBlock in order to indicate the possible exceptions thrown.

Reply:
Hi @dmytro-ch : I ignore the rule because the Static Test doesn't allow . "ThrowCatch Exceptions must not be handled in the same function". So could I remove throw exception and change to `$message = ` . But I aware it can affect to other file can call it. So I prevent it and mark it ignore to pass Static Test.